Python-packages/covidcast-py/docs/getting_started.rst

```diff
@@ -104,6 +104,15 @@ May 1st were updated on May 23rd based on new data, giving a ``lag`` of 22 days.
 See the :py:func:`covidcast.signal` documentation for details on the returned
 data frame.
 
+By default, this package submits queries to the API anonymously. If you have an
+API key, you can use it with this package by calling
+:py:func:`covidcast.use_api_key`, then call fetch functions as normal:
```

Comment:
This will cross-reference with a link to the `covidcast.use_api_key` docs, but those docs aren't actually included in Sphinx anywhere. Probably we want to add an "API keys" section to `signals.rst`, containing `.. autofunction: covidcast.use_api_key`, right?

Reply:
Done